helix-term/src/health.rs

```diff
@@ -268,6 +269,13 @@ pub fn language(lang_str: String) -> std::io::Result<()> {
         }
     };
 
+    if display_heading {
+        let stdout = std::io::stdout();
+        let mut stdout = stdout.lock();
+
+        writeln!(stdout, "\n{}", &lang_str.clone().white())?;
+    }
```

Comment:
```suggestion
    if display_heading {
        writeln!(stdout, "\n{}", &lang_str)?;
    }
```

1 - `stdout` is already locked at the beginning of the function. 2 - I think we shouldn't colorize this text so that it is visible even on a light background. Same as here https://github.com/helix-editor/helix/blob/f19793c2f86dea8e7af8570563e2dd5a8da9fefb/helix-term/src/health.rs#L303

Reply:
Ok, fixing those asap. Would it be useful to add some sort of styling (like underlining) to the heading?

Reply:
I would like the first letter to be capitalized, but that is a bit tricky to do. Underlining sounds good to me. Also, I was wrong about `.white()`, it looks like it actually goes `black` on a white background, so it's even better to left it, xd.
`.white()`:
![image](https://github.com/helix-editor/helix/assets/122799969/b6bdb12d-df63-4071-94a5-78141c1f6c66)
vs `plain`:
![image](https://github.com/helix-editor/helix/assets/122799969/f94c4a08-9678-4b61-84ec-81e209c1da8d)

Reply:
Reverting heading style :P. 
Don't know if the underline part would be ok, because it wouldn't match the heading of `--health langugages`.

Could be implemented in another pr.

Reply:
What do you mean "woudln't match"? It's just a visual effect for `--health <lang>`, it won't affect table or what?

Reply:
the table's heading are just white and aren't underlined.
For consistency I think we should underline both table headings and single language headings or none

Reply:
Gotcha, LGTM (my vote doesn't decide anything anyway :smiley:)